src/routes/v1/challenge.route.js

```diff
@@ -2,43 +2,39 @@ const express = require("express");
 const router = express.Router();
 const challengeController = require("../../controllers/challenge.controller");
 
-router
-  .route("/")
-  .get((req, res) => challengeController.getRandomChallenge(req, res));
-
-router
-  .route("/:id")
-  .get((req, res) => challengeController.getChallengeById(req, res));
```

Comment:
Why removed?

Reply:
Changed name of getRandomChallenge to get Challenge because the parameter check occurs within the function.

I got right of line 9 because the issue said it wanted a query paramter rather than a router